src/pages/EnablePayments/ActivateStep.js

```diff
@@ -21,24 +27,64 @@ const defaultProps = {
     userWallet: {},
 };
 
-const ActivateStep = props => (
-    <ScreenWrapper>
-        <HeaderWithCloseButton
-            title={props.translate('activateStep.headerTitle')}
-            onCloseButtonPress={() => Navigation.dismissModal()}
-            shouldShowBackButton
-            onBackButtonPress={() => Navigation.goBack()}
-        />
-        <View style={[styles.mh5, styles.flex1]}>
-            {props.userWallet.tierName === CONST.WALLET.TIER_NAME.GOLD && (
-                <Text>{props.translate('activateStep.activated')}</Text>
-            )}
-            {props.userWallet.tierName === CONST.WALLET.TIER_NAME.SILVER && (
-                <Text>{props.translate('activateStep.checkBackLater')}</Text>
-            )}
-        </View>
-    </ScreenWrapper>
-);
+class ActivateStep extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.renderGoldWalletActivationStep = this.renderGoldWalletActivationStep.bind(this);
+    }
+
+    renderGoldWalletActivationStep() {
+        return (
+            <>
+                <View style={[styles.pageWrapper, styles.flex1, styles.flexColumn, styles.alignItemsCenter, styles.justifyContentCenter]}>
+                    <Icon
+                        src={Illustrations.TadaBlue}
+                        height={100}
+                        width={100}
+                        fill={defaultTheme.iconSuccessFill}
+                    />
+                    <View style={[styles.ph5]}>
+                        <Text style={[styles.mt5, styles.h1, styles.textAlignCenter]}>
+                            {this.props.translate('activateStep.activatedTitle')}
+                        </Text>
+                        <Text style={[styles.mt3, styles.textAlignCenter]}>
+                            {this.props.translate('activateStep.activatedMessage')}
+                        </Text>
+                    </View>
+                </View>
+                <FixedFooter>
+                    <Button
+                        text={this.props.translate('common.continue')}
```

Comment:
Not a big deal but we can probably adjust this now or in a follow up PR.

```
The green success button will either state “Continue to payment” or “Continue to transfer” based on the action that we will be returning the user to.
```

https://docs.google.com/document/d/1PYqKcBpjPxewL5OneCmqbMJNH8G3CNkqbJWqzHSaXhU/edit#bookmark=id.8y4wjrbvjh9z

Reply:
Yeah at the moment I don't see how we would do that so I don't mind if we do this in a follow-up PR.